packages/template-retail-react-app/config/default.js

```diff
@@ -36,6 +36,17 @@ module.exports = {
             // This differs from the siteId in commerceAPIConfig for testing purposes
             siteId: 'aaij-MobileFirst',
             isProduction: false
+        },
+        // getAppOrigin will take x-forwarded-host header into account if this is true
+        useXForwardedHost: false,
+        login: {
+            passwordless: {
+                enabled: false
+            },
+            social: {
+                enabled: false,
+                idps: ['google', 'apple']
+            }
```

Comment:
made the default to have passwordless and social disabled in the retail react app but the out-of-the-box supported socials are listed